facade/service.go

```diff
@@ -713,27 +726,30 @@ func (f *Facade) validateServiceMigration(ctx datastore.Context, svcs []service.
 
 func (f *Facade) RemoveService(ctx datastore.Context, id string) error {
 	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.RemoveService"))
+	alog := f.auditLogger.Message( ctx, "Removing Service" ).
+		Action(audit.Remove).ID(id)
```

Comment:
Please add `.Type(service.GetType())` where GetType() is a new method you need to add to the Service domain object similar to what @corntoole added for the resource pool domain object